tests/emails/beneficiary_booking_confirmation_test.py

```diff
@@ -0,0 +1,214 @@
+from datetime import datetime, timezone
+from unittest.mock import patch
+
+from emails.beneficiary_booking_confirmation import retrieve_data_for_user_booking_confirmation_email
+from models import PcObject
+from tests.conftest import clean_database
+from tests.test_utils import create_offerer, create_venue, create_user, create_booking, create_offer_with_event_product, \
+    create_stock_from_offer, create_deposit, create_offer_with_thing_product, create_mediation
+
+
+@clean_database
+def test_should_return_event_specific_data_for_email_when_offer_is_an_event(app):
+    # Given
+    user = create_user(first_name='Joe', email='joe@example.com')
+    deposit = create_deposit(user)
+    offerer = create_offerer(idx=1, name='Théâtre du coin')
+    venue = create_venue(offerer, 'Lieu de l\'offreur', idx=1,
```

Comment:
```suggestion
    venue = create_venue(offerer, "Lieu de l'offreur", idx=1,
```
C'est plus simple si on veut faire une recherche dessus.